.buildkite/scripts/unset-secrets.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+# Unset all variables ending with _SECRET
+for var in $(printenv | sed 's;=.*;;' | sort); do
+  if [[ "$var" == *_SECRET || "$var" == *_TOKEN]]; then
```

Comment:
Missing space ?
```suggestion
  if [[ "$var" == *_SECRET || "$var" == *_TOKEN ]]; then
```
In my laptop, this fails:
```shell
 $ if [[ "$var" == *_SECRET || "$var" == *_TOKEN]]; then echo "deleted"; else echo "don't"; fi
bash: syntax error in conditional expression: unexpected token `;'
bash: syntax error near `;'
```